app/models/application_form.rb

```diff
@@ -209,9 +209,7 @@ def candidate_has_previously_applied?
   end
 
   def candidate_can_choose_single_course?
-    return false if FeatureFlag.active?(:apply_again_with_three_choices)
-
-    apply_2?
+    false
```

Comment:
Drop this whole method?

Reply:
Thanks Frankie, i was originally going to refactor in a seperate PR but i've just done it here.